ironic-config/apache2-ironic-api.conf.j2

```diff
@@ -36,17 +36,27 @@ Listen 6385
     SSLCertificateKeyFile {{ env.IRONIC_KEY_FILE }}
 {% endif %}
 
+    <Location /v1/lookup >
+        Require all granted
+    </Location>
+    <Location /v1/heartbeat/ >
```

Comment:
I wonder if the trailing `/` is an issue here. I see HTTP 401 when using `/v1/heartbeat`